packages/contracts-bedrock/scripts/upgrades/stage-1.4/README.md

```diff
@@ -0,0 +1,38 @@
+# Stage 1.4 Upgrade
+
+This directory contains a repeatable task for deploying and registering the stage 1.4 `FaultDisputeGame` with a
+`DisputeGameFactory` contract.
+
+## Dependencies
+
+- [`foundry`](https://getfoundry.sh/)
+- [`golang`](https://go.dev/doc/install)
+- [`jq`](https://jqlang.github.io/jq/download/)
+
+## Usage
+
+```sh
+# 1. Clone the monorepo, checkout the `op-contracts/v1.9.0-rc.3` tag, and navigate to this directory.
+git clone --branch op-contracts/v1.9.0-rc.3 git@github.com:ethereum-optimism/optimism.git && \
+  cd optimism/packages/contracts-bedrock/scripts/upgrades/stage-1.4
+
+# 2. Set up the `.env` file
+cp .env.example .env && vim .env
+
+# 3. Run the deployment task.
+#
+# This task will deploy a new `FaultDisputeGame` + `RISCV` VM + `DelayedWETH` proxy for the L2 chain that the
+# deploy config configures.
+#
+# The dispute game will be configured per the deploy config that is configured in `.env`, with some exceptions:
+# 1. The game type is set to `3`, or the `ASTERISC_KONA` game type.
+# 2. The absolute prestate is pulled from the kona release used for `op-contracts/v1.9.0-rc.3`
+#    (`kona-client-v0.1.0-beta.5`)
```

Comment:
We probably shouldn't hard code this as it will need to vary for different chains depending on when their latest config is updated in kona.